packages/api/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export { BrevoTransactionalMailsService } from "./brevo-api/brevo-api-transactional-mails.service";
+export { BrevoContactImportService } from "./brevo-contact/brevo-contact-import.service";
```

Comment:
I'm not sure if it makes sense exporting the importService, when it is intended to use either the .controller or the .console

What might be the use case for exporting the Import Service?

I can only think of a custom importer console or controller, but I am not sure why this might be necessary.

Reply:
2a661f004d4f63139d49fc34ddc5d67da59f588b

Then we'll not export it for now